main.py

```diff
@@ -1,27 +1,47 @@
 """UI Server, Connects all Components."""
-
+import os
+from file_adapter import FileAdapter
 from flask import Flask
 from flask import make_response
 from flask_restful import Api
+import resources
+from resources import Releases
 from resources import Resources
-
+from airflow_connector import AirflowDB
 
 # creating the Flask application
 APP = Flask(__name__)
 API = Api(APP)
+resources = Resources()
+
 
-REST = Resources()
+adapter = FileAdapter('fake_data/fake_release_data.json', 'fake_data/fake_task_data.json')
+cloudsql_unix_socket = os.path.join('/cloudsql', 'istio-release-ui:us-central1:prod-airflow-snapshot-sandbox')
+cloudsql_host = '35.193.234.53'
+cloudsql_user = 'root'
+cloudsql_password = ''
+cloudsql_db = 'airflow-db'
+airflow_db = AirflowDB(unix_socket=cloudsql_unix_socket,
+                       host=cloudsql_host,
+                       user=cloudsql_user,
+                       password=cloudsql_password,
+                       db=cloudsql_db)
+
+
+my_resources = Releases(adapter)
 
 # adding resource endpoints to different urls
-API.add_resource(REST.releases, '/releases')
-API.add_resource(REST.release, '/release')
-API.add_resource(REST.branches, '/branches')
-API.add_resource(REST.types, '/types')
-API.add_resource(REST.tasks, '/tasks')
+API.add_resource(resources.releases, '/releases', resource_class_kwargs={'adapter': adapter})
```

Comment:
I think you meants resource.Releases

Reply:
It actually doesn't work if it is capitalized, since it refers to the Resources class in resources.py. If you want, I can change it to capitals though.